src/Repository/UserRepository.php

```diff
@@ -7,5 +7,12 @@
 
 class UserRepository extends AbstractRepository
 {
```

Comment:
This method return type hint tells me that you return an array. Later on you use the value as a number (in the template) smth is weird.